### DISCUSSION TITLE
Anyway to have casks & formula one tap?

### DISCUSSION THREAD

**Discussion by Gcenx** [Homebrew/discussions#2946] [Tap maintenance and Homebrew development] [answered]
The tap contains some casks but I also wanted to start to add some formulas, fixed the syntax for the casks but still fails later
https://github.com/Gcenx/homebrew-wine/runs/5171409290

I don’t see anything on homebrew-cask using the syntax it’s deciding to now fail on.

Is this intentional for casks/formula to be on separate taps or the tests don’t account for this/I’m missing something?

**Comment by carlocab** [comment, marked as answer]
We don't use `test-bot` in `homebrew-cask[-*]` taps, so I think this is just an oversight.

You can probably do `brew style --fix Gcenx/wine` to autocorrect those errors. In principle there shouldn't be any barriers to having a tap that contains casks in the `Casks` subdirectory and formulae in the `Formula` subdirectory.

**Reply by Gcenx** [reply]
Thanks after running that and doing a couple of manual tweaks the added formula is now building.
